core/templates/dev/head/player/conversation_skin_directive.html

```diff
@@ -214,7 +214,8 @@
       mode.
     */
     html, body {
-      background: url('/images/general/background.jpg') no-repeat center center fixed;
+      /*TODO(gvishal): modify image url to use cache slugs*/
```

Comment:
Actually -- would it be prudent to give /assets/common a fairly short cache time, if we're going with that for now?